en/attributes.html

```diff
@@ -123,10 +141,19 @@ <h2 id="data-structures">Data structures</h2>
       One large memory buffer holding references (32 bit) to a memory structure
       with objects per unique value.
     </td>
+  </tr><tr>
+    <td>Tensor</td>
+    <td>Fixed / Variable</td>
+    <td>
+      Like the "D" attribute - a tensor is a multivalued datatype with sparse and/or dense dimensions,
+      stored in the <em>tensor store</em>.
+      Dense dimensions resemble the attribute multivalue mapping, as elements are fixed in size and numbers.
```

Comment:
I would skip the comparison with the other data stores, as I find it a bit misleading. Consider the following simplification instead: "The memory layout in the tensor store depends on the tensor type".

Reply:
ok!